app.js

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import { config } from './config/index.js';
+import meteorsController from './delivery/meteorsController.js';
+
+const app = express();
+
+app.get('/', (req, res) => {
+  res.send('Server is running');
+});
+
+app.use(meteorsController);
+
```

Comment:
add `errorHandler` please somewhere there

here you can find a little bit more information https://medium.com/@mohsinogen/simplified-guide-to-setting-up-a-global-error-handler-in-express-js-daf8dd640b69